addons/point_of_sale/wizard/pos_confirmation_wizard.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_confirm_action_wizard" model="ir.ui.view">
```

Comment:
```suggestion
    <record id="point_of_sale.view_confirm_action_wizard" model="ir.ui.view">
```

prefix even if it's implicitly in pos. It's easier to find this view with the prefix.